pkg/datastore/postgres/repositories/watched_events_repository_test.go

```diff
@@ -79,7 +79,9 @@ var _ = Describe("Watched Events Repository", func() {
 		Expect(err).ToNot(HaveOccurred())
 		blockId, err := blocksRepository.CreateOrUpdateBlock(core.Block{})
 		Expect(err).NotTo(HaveOccurred())
-		receiptId, err := receiptRepository.CreateReceipt(blockId, core.Receipt{})
+		tx, _ := db.Beginx()
```

Comment:
Worth assigning and asserting against an error here?

Reply:
07e45dda2ca21f2fe3b548f28cb7fc3bd99fdc9b